src/arrayMethodSort.js

```diff
@@ -4,8 +4,38 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  function basicCompare(a, b) {
+    const stringA = String(a);
+    const stringB = String(b);
+
+    if (stringA > stringB) {
+      return 1;
+    } else if (stringA === stringB) {
+      return 0;
+    } else {
+      return -1;
+    }
+  }
```

Comment:
When we write `return` inside `if` - `else` is redundant.
```suggestion
    if (stringA > stringB) {
      return 1;
    } 
    if (stringA === stringB) {
      return 0;
    }
    
    return -1;
  }
```

Reply:
I somehow missed that. That's obvious. Thanks!